docs/articles/expensify-classic/bank-accounts-and-payments/payments/Enable-Global-Reimbursement.md

```diff
@@ -0,0 +1,73 @@
+
+**Note:** Each country has specific identity verification, compliance, and onboarding requirements. Use the guides above to ensure you're submitting the correct documentation.
+
+---
+
+# Requirements
+
+| Country         | Required Documents                                                                 | Direct Debit Setup Time             | Special Requirements | Test Debits Sent |
+|----------------|--------------------------------------------------------------------------------------|-------------------------------------|----------------------|------------------|
+| **U.S.**        | Partner form, address verification, bank statement                                 | ⏳ 1-2 business days post-verification | None                | Yes              |
+| **Canada**      | Partner form, address verification, bank statement, proof of beneficial ownership  | ⏳ 1-2 business days post-verification | None              | Yes              |
+| **UK**          | Partner form, address verification, bank statement, proof of beneficial ownership, ID for signatories | ⏳ 1-2 weeks post-verification | None                | Yes              |
+| **EU**          | Partner form, address verification, bank statement, proof of beneficial ownership, certified ID copies | ⏳ 1-2 weeks post-verification | <br>🇮🇹 **Italy:** Requires Tax ID<br>🇪🇸 **Spain:** No driver's license as ID | Yes              |
+| **Australia**   | Partner form, address verification, bank statement, proof of beneficial ownership, ID for signatories | ⏳ 1-2 business days post-verification | None              | Yes              |
+| **Singapore**   | Partner form, address verification, proof of beneficial ownership, board resolution, memorandum & articles (M&A), certified ID copies | ⏳ ~3 weeks post-verification | <br>💡Pre-funding recommended | Yes               |
+
+
+---
+
+# FAQ
+
+## Can other admins send global reimbursements?
+Yes. Once verified, the reimbursement account can be shared with other workspace admins.
+
+## How long does it take to verify an account for global reimbursement?
+The verification process can take anywhere from a few business days to several weeks, depending on the information provided in the DocuSign form and if additional information is required for compliance.
+
+## Who is the "Authorized User" on the DocuSign form?
+This is the person who manages reimbursements and bank setup in Expensify.
+
+## Why can't my employee add a deposit account?
+Make sure your employee's default workspace is set to the one linked to the bank you're using for global payments. They can confirm their default workspace by following [these steps](https://help.expensify.com/articles/expensify-classic/workspaces/Navigate-multiple-workspaces).
+
+## Can I connect my Canadian bank account if the currency of the account is in USD? 
+Yes, since USD is considered a local currency in Canada, you use a Canadian bank account that uses USD for global reimbursements. 
```

Comment:
```suggestion
Yes, since USD is considered a local currency in Canada, you can use a Canadian bank account that uses USD for global reimbursements. 
```